### DISCUSSION TITLE
NEED HELP

### DISCUSSION THREAD

**Discussion by BallisticOK** [TGRHavoc/live_map-interface#55] [Q&A] [answered]
hey i download and follow all the steps to the point and the web map cant see the server
i get this error ever time
![image](https://user-images.githubusercontent.com/39306849/111025698-fd251400-8439-11eb-8045-ccb9abb6e21c.png)

**Comment by TGRHavoc** [comment, marked as answer]
See https://github.com/TGRHavoc/live_map-interface/issues/43
